README.md

```diff
@@ -102,51 +102,43 @@ Alternatively, under root:
 
 You can add "-d" to build_full_frontend.sh to disable obfuscation and reduce build time.
 
-### Install and run pushpin for UI updates
+### Run Tracim for development
 
-Tracim relies on [pushpin](https://pushpin.org/docs/about/) for UI updates, so you need to setup pushpin.
+    ./run_dev_backend.sh
 
-#### For development purpose
+This script will launch the minimal services for a functional backend: pushpin + web application.
```

Comment:
We should warn user that this does use docker and docker compose stuff.

Reply:
Right, I'll add this.